less/_components/quicksilver/dark-theme.less

```diff
@@ -0,0 +1,13 @@
+.prefers-color-scheme{
+    @media (prefers-color-scheme: dark) {  
+        body footer{
```

Comment:
did you mean body, footer?

Reply:
@chrisguindon Yeah you are right, sorry for that.